tracee-ebpf/tracee/tracee.go

```diff
@@ -1345,13 +1345,13 @@ func (t *Tracee) processFileWrites() {
 				continue
 			}
 			if appendFile {
-				if _, err := f.Seek(0, os.SEEK_END); err != nil {
+				if _, err := f.Seek(0, io.SeekEnd); err != nil {
 					f.Close()
 					t.handleError(err)
 					continue
 				}
 			} else {
-				if _, err := f.Seek(int64(meta.Off), os.SEEK_SET); err != nil {
+				if _, err := f.Seek(int64(meta.Off), io.SeekStart); err != nil {
```

Comment:
This needs to be seek_set so tracee can rebuild the file according to the correct offsets. If you set it to seekstart instead, it will overwrite other data written, and the written file will be corrupted

Reply:
@yanivagman `os.SEEK_SET` is equivalent to `io.SeekStart`. This change is merely using the same value from a different package (which is good)

Reply:
Correct. My mistake.